lmdeploy/turbomind/deploy/target_model/base.py

```diff
@@ -136,26 +136,34 @@ def export_config(self) -> None:
 
     def export_weight(self, param: torch.Tensor, name: str) -> None:
         """export turbomind weight."""
+
+        def _tofile(tensor, path):
+            """to file."""
+            if tensor.dtype == torch.bfloat16:
+                tensor = tensor.view(torch.half)
```

Comment:
Can a tensor view `datatype`?

Reply:
https://pytorch.org/docs/stable/generated/torch.Tensor.view.html

It is a python version reinterpret cast.
numpy does not support bf16